bot/src/infrastructure/mapper/PrismaMapper.ts

```
@@ -4,14 +4,48 @@ import { ClassConstructor, plainToInstance } from "class-transformer";
+  }
+
+  /**
+   * Recursively converts an object's keys from snake_case to camelCase
+   */
+  private static convertKeysToCamelCase(obj: any): any {
+    if (obj === null || obj === undefined) {
+      return obj;
+    }
+
+    if (Array.isArray(obj)) {
+      return obj.map((item) => this.convertKeysToCamelCase(item));
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)